.sbt-java-formatter.conf

```diff
@@ -8,6 +8,11 @@ ignored-files = [
   "StreamRefMessages.java",
   //in tests
   "ProtobufProtocol.java"
+  //formtat imports cause errors
+  "AggregatorTest.java",
+  "InteractionPatternsTest.java",
+  "ReplicatedShardingTest.java",
+  "ReplicatedAuctionExampleTest.java"
```

Comment:
I add thoese files to ignore list.

Reply:
Have not tried the https://github.com/google/google-java-format/issues/137#issuecomment-567714532 yet

Reply:
Fair enough. 

We correctly using spotless for java formatting, it has configable than raw `google-java-formatter`.
and then we meet the same issue of turn off the snippet formatting. In the end, we using new feature of spotless plugin, it sovled my problem. 

There is some context of this feature and issue: https://github.com/diffplug/spotless/issues/275

sbt has a plugin for spotless, but it has not been updated for a long time.

Reply:
I just verified if this issue can be fixed using `spotless`, the answer is no, there is exception i have meet, i am very sure it will break the `paradox`.

```java
com.google.googlejavaformat.java.FormatterException: error: Imports not contiguous (perhaps a comment separates them?)
```